JenkinsfileRT

```diff
@@ -21,7 +21,8 @@ bc.nodetype = 'linux'
 bc.env_vars = test_env
 bc.name = '3.6'
 bc.conda_channels = ['http://ssb.stsci.edu/astroconda']
-bc.conda_packages = ['numpy',
+bc.conda_packages = ['acstools',
+                     'numpy',
```

Comment:
I was wondering whether simply setting this to 'numpy < 1.16' would fix the problems with the testing on Boyle...